pkg/kubeapiserver/options/plugins.go

```diff
@@ -20,9 +20,6 @@ package options
 // This should probably be part of some configuration fed into the build for a
 // given binary target.
 import (
-	// Cloud providers
-	_ "k8s.io/kubernetes/pkg/cloudprovider/providers"
```

Comment:
where is this imported for kube-apiserver? is it also imported properly for hyperkube?

Reply:
It's imported in cmd/kube-apiserver/app/options in the kube-apiserver. Hyperkube still gets it from cmd/{controller-manager,kubelet,kube-apiserver}/app.

Before:

![before](https://user-images.githubusercontent.com/1787169/53276697-10c77b80-36b5-11e9-9c74-c58401c78ee3.png)


After:

![after](https://user-images.githubusercontent.com/1787169/53276698-12913f00-36b5-11e9-8295-c765018ae147.png)